spk/octoprint/src/config.yaml

```diff
@@ -3,3 +3,13 @@ serial:
 
 server:
   port: 8088
+
+plugins:
+  cura:
+    cura_engine: /usr/local/curaengine/bin/CuraEngine
```

Comment:
I assume this doesn't cause issues if CuraEngine is not installed? E.g. error messages in logs or things like that?